packages/react-link/src/components/Link/useLink.ts

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import { useMergedRefs } from '@fluentui/react-hooks';
+import { makeMergeProps, resolveShorthandProps } from '@fluentui/react-utils';
+import { LinkProps, LinkState } from './Link.types';
+import { useLinkState } from './useLinkState';
+
+/**
+ * Consts listing which props are shorthand props.
+ */
+export const linkShorthandProps = [];
```

Comment:
is this unnecessary ?

Reply:
No, I pass it down in line 24 (you need to pass at least an empty array). It also helps having it defined like that in case it needs to grow since it is being used anyway.

Reply:
that's fair :-)